entproto/internal/todo/ent/proto/entpb/entpb.proto

```diff
@@ -316,7 +316,7 @@ message Todo {
 }
 
 message User {
-  int64 id = 1;
+  uint32 id = 1;
```

Comment:
@vietdoan doesnt this mean that this is a breaking change for all existing projects?

Reply:
no, it's not. int32, uint32, int64, uint64, and bool are all compatible according to https://developers.google.com/protocol-buffers/docs/proto3

Reply:
Yes I'm aware of that. This pertains to the proto wire format. The generated code will not be compatible though and will require changes from consumers. I'd like to avoid this if possible

Reply:
@rotemtam I see! If they update dependencies, they inevitably have to manually update their code. However, isn't error during compile time not a big deal? Old consumers (using old protobuf schema) are still able to communicate with producers/servers (using new generated code) due to the compatibility mentioned above.